test/module.spec.js

```diff
@@ -1,30 +1,63 @@
 import { expect } from 'chai';
-import { findModules, compileModules } from '../src/module';
+import {
+    compileModules,
+    evaluateModules,
+    findAndCompileModules ,
+    findModules
+} from '../src/module';
+import { createVM } from '../src/vm';
 
 describe('module.js', () => {
     describe('Module compilation', () => {
         it('should list all JS and JSON files', () => {
             return findModules()
                 .then(files => {
                     expect(files).to.be.an('array');
-                    expect(files).to.have.lengthOf(1);
                 });
         });
 
         it('should correctly compile a JSON module', () => {
             return compileModules(['package.json'])
                 .then(compiledFiles => {
-                    expect(compiledFiles).to.be.an('object');
-                    expect(compiledFiles).to.have.property('package.json');
-                    expect(() => JSON.parse(compiledFiles['package.json'])).to.not.throw(Error);
+                    expect(compiledFiles).to.be.an('array');
+                    expect(compiledFiles).to.have.length(1);
                 });
         });
 
         it('should correctly compile a JS module', () => {
             return compileModules(['./test/module.spec.js', './test/build.spec.js'])
                 .then(compiledFiles => {
-                    expect(compiledFiles).to.be.an('object');
+                    expect(compiledFiles).to.be.an('array');
                 });
         });
+
+        it('should bootstrap the rung-cli project!!!', () => {
+            return findAndCompileModules()
+                .then(results => {
+                    expect(results).to.be.an('array');
```

Comment:
The test is very basic, it is true. If it reaches there, the bootstrap was successful, although, because contextualized JS code will always be returned in this situation. The biggest proof is that the promise is _resolved_, and there are other tests that cover this resolution. This looks more like an exhaustive proof.